kratos/utilities/variable_utils.h

```diff
@@ -721,6 +739,23 @@ class KRATOS_API(KRATOS_CORE) VariableUtils
         KRATOS_CATCH("")
     }
 
+    /**
+     * @brief Set the Historical Variables To Zero
+     * This method sets the provided list of variables to zero in the nodal historical database
+     * @tparam TVariableArgs Variadic template argument representing the variables types
+     * @param rNodes Nodes container to be initialized to zero
+     * @param rVariableArgs Variables to be set to zero
+     */
+    template<class... TVariableArgs>
+    static void SetHistoricalVariablesToZero(
+        NodesContainerType& rNodes,
+        const TVariableArgs&... rVariableArgs)
+    {
+        block_for_each(rNodes, [&](NodeType& rNode){(
+            AuxiliaryHistoricalValueSetter<typename TVariableArgs::Type>(rVariableArgs, rVariableArgs.Zero(), rNode), ...);
```

Comment:
```suggestion
            (rNode.FastGetSolutionStepValue(rVariableArgs) = rVariableArgs.Zero(), ...);
```

Wouldn't this work without any problems since `rNode.FastGetSolutionStepValue(rVariableArgs)` anyways returns the reference? Do we still need the `noalias`?

Reply:
I did the auxiliary method to enable using `noalias` with the `array_1d`. This would work as well but wouldn't be as efficient.